devicetypes/smartthings/smartsense-multi-sensor.src/smartsense-multi-sensor.groovy

```diff
@@ -461,19 +489,34 @@ private Map parseAxis(String description) {
 			xyzResults.x = signedX
             log.debug "X Part: ${signedX}"
         }
+        // Y and the Z axes are interchanged between SmartThings's implementation and Centralite's implementation
         else if (part.startsWith("13")) {
 			def unsignedY = hexToInt(part.split("13")[1].trim())
 			def signedY = unsignedY > 32767 ? unsignedY - 65536 : unsignedY
-			xyzResults.y = signedY
-            log.debug "Y Part: ${signedY}"
+			if (device.getDataValue("manufacturer") == "SmartThings") {
+                xyzResults.z = signedY
```

Comment:
Based on my testing, `signedY` needs to be negated in order to match the Centralite implementation. As it is currently implemented, when the Z value is positive on Centralite it is negative on the other one.